pkgs/by-name/be/beidconnect/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pcsclite,
+  boost,
+  pkg-config,
+  testers,
+  beidconnect,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "beidconnect";
+  version = "2.10";
+
+  src = fetchFromGitHub {
+    owner = "Fedict";
+    repo = "fts-beidconnect";
+    rev = finalAttrs.version;
+    hash = "sha256-xkBldXOlgLMgrvzm7ajXzJ92mpXrxHD1RX4DeBxU3kk=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    pcsclite.dev
+    boost
+  ];
+
+  strictDeps = true;
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-fail '$(DESTDIR)/usr/bin' '$(DESTDIR)/bin'
+  '';
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+  sourceRoot = "${finalAttrs.src.name}/linux";
+
+  postInstall = ''
+    install -d $out/webkit-hosts $out/lib/mozilla/native-messaging-hosts
+
+    $out/bin/beidconnect -setup $out/bin $out/webkit-hosts $out/lib/mozilla/native-messaging-hosts
+  '';
```

Comment:
Regardless of the NixOS integration, these should go to the proper FHS paths as before rather than into `$out/webkit-hosts`. (BTW, Chromium doesn’t use WebKit any more and this format is specific to Chromium rather than the underlying Blink engine, so the path is pretty misleading anyway :) )